### DISCUSSION TITLE
deleting win11 default wsa packages folder path contents if using wsa with magisk and gapps

### DISCUSSION THREAD

**Discussion by myusrn** [LSPosed/MagiskOnWSALocal#202] [Q&A] [answered]
Once one is successfully running the wsa with magisk and mindthegapps environment, e.g. current WSA-with-magisk-stable-MindTheGapps_2209.40000.26.0_x64_Release-Nightly build out of a C:\wsaMagiskGapps path, is the appx 7GB+ of content found in the $env:localappdata\Packages\MicrosoftCorporationII.WindowsSubsystemForAndroid_8wekyb3d8bbwe path still necessary?

In other words if one would like to recover that disk space is it save to scrub the contents of that folder and the folder itself?

**Comment by Howard20181** [comment, marked as answer]
This place is used to save the data of WSA that needs to be kept
MagiskOnWSA has not changed the package name, so it uses the same data directory as the original version

**Reply by myusrn** [reply]
Thanks for the response.  Is the "wsa with magisk and mindthegapps" folder that i executed .\Run.bat [ -> Install.ps1 ] from which has to be kept around overriding some aspects of that default wsa package location and data or modifying so as to point to additions that are kept in the "wsa with magisk and mindthegaps" folder?

**Reply by Howard20181** [reply]
The folder where you executed `Run.bat` saves the WSA system files, `%LOCALAPPDATA%\Packages\MicrosoftCorporationII.WindowsSubsystemForAndroid_8wekyb3d8bbwe` saves the WSA Android user data.

**Reply by Howard20181** [reply]
Deleting `%LOCALAPPDATA%\Packages\MicrosoftCorporationII.WindowsSubsystemForAndroid_8wekyb3d8bbwe\LocalCache\userdata.vhdx` is equivalent to executing factory reset on a phone
